ImageAttachment.ts

```diff
@@ -0,0 +1,15 @@
+import { IHttp, IModify, IPersistence, IRead, IMessageExtender } from '@rocket.chat/apps-engine/definition/accessors';
+import { IMessageAttachment } from '@rocket.chat/apps-engine/definition/messages';
+
+export class ImageAttachment implements IMessageAttachment{
+    imageUrl?: string;
+
+    constructor(imgUrl: string){
+        this.imageUrl = imgUrl;
+    }
+
+    private setImage(imgUrl: string){

```

Comment:
This method is not being used, so it would be nice to remove it

Reply:
Good suggestion! I've just removed it.